experimental/algorithm/LAGraph_msf.c

```diff
@@ -96,15 +98,34 @@ bool f1 (
     return (weight[i] == *aij) && (parent[j] == partner[i]);
 }
 
+#else
+// for GrB_IndexUnaryOp:
+void f1 (bool *z, const void *x, GrB_Index i, GrB_Index j, const void *thunk)
+{
+    uint64_t *aij = (uint64_t*) x;
+    (*z) = (weight[i] == *aij) && (parent[j] == partner[i]);
+}
+#endif
+
 // edge removal:
 // A(i, j) is removed when parent[i] == parent[j]
+#if 0
+// for GxB_SelectOp:
 bool f2 (
     GrB_Index i, GrB_Index j,
     const void *x, const void *thunk)
 {
     uint64_t *aij = (uint64_t*) x;
     return (parent[i] != parent[j]);
 }
+#else
+// for GrB_IndexUnaryOp:
+void f2 (bool *z, const void *x, GrB_Index i, GrB_Index j, const void *thunk)
+{
+    uint64_t *aij = (uint64_t*) x;
```

Comment:
If this line is only to avoid compiler warnings (a noble cause), I think there is another cleaner way:  I used to be able to just remove the parameter name 'x' from the function's parameter list.

Reply:
Unsure.  I was just revising the prior operator and didn't notice that aij was unused.  That line should be removed.